include/experimental/group/gemm/impl/int4_dequantize_xmx_xe.hpp

```diff
@@ -398,19 +406,24 @@ class gemm_t<
                     matB, matB_payload);
             subgroup::tile_load<cache_hint::cached, cache_hint::cached>(
                     scale, scale_payload);
-            subgroup::tile_load<cache_hint::cached, cache_hint::cached>(
-                    zero_pt, zero_pt_payload);
+            if constexpr (compute_policy::quant_type == quant_mode::S4_CLIP) {
```

Comment:
in fact, jblas suppor asym+s4_fullrange config(but qbits only expose sym+bitN feature to user), so it's ok to load zp when quant_mode==fullrange(if user choose asym quant alg) in xetla level

Reply:
Maybe I should change it to s4_fullrange_no_zp